docs/reference/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -10,5 +10,14 @@ coming[7.2.0]
 //Installation and Upgrade Guide
 
 // tag::notable-highlights[]
+[discrete]
+==== {dataframes-cap}
+
+beta[] You can now transform your data with
+{stack-ov}/ml-dataframes.html[data frames]. There is a new {kib} wizard that
+guides you through the process of creating a {dataframe-transform} to pivot and
+summarize your data and store it in a new index. Alternatively, you can use
+{ref}/data-frame-apis.html[{dataframe} APIs] to preview, create, and maintain
```

Comment:
```suggestion
{ref}/data-frame-apis.html[{dataframe} APIs] to preview, create, and administrate
```
Administrate seems better, but the grammar does not quite line up...maintain still works just fine.

Reply:
What about "manage"?

Reply:
@lcawl manage is good 👍